<launch>
    <include file="$(find kimera_interfacer)/launch/labdata.launch" />

    <node name="label_generator" pkg="label_generator_ros" type="ray_cast_scene.py" output="screen" />

    <node name="mocked_control_node" pkg="control_node" type="Control_mock.py" output="screen">
        <param name="image_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/color" />
        <param name="depth_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/depth" />
        <param name="gt_label_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/label_nyu40" />
        <param name="dlab_label_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/deeplab_labels" />
        <param name="pose_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/pose" />
        <param name="mesh_path" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Pipeline/Output_kimera_mesh/output_predict_mesh.ply" />
        <param name="serialized_path" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Pipeline/Output_kimera_mesh/output_serialized.data" />
        <param name="mapping_file" value="$(find control_node)/cfg/nyu40_segmentation_mapping.csv" />
        <param name="intrinsic_dir" value="/home/michele/Desktop/Domain-Adaptation-Pipeline/IO_pipeline/Scannet/scans/scene0002_00/intrinsic" />
    </node>
</launch>
